testing/tests/DevExpress.ui.widgets.pivotGrid/pivotGrid.tests.js

```diff
@@ -2994,6 +2994,126 @@ QUnit.module('dxPivotGrid', {
         scrollable.scrollTo({ left: 10, top: 1 });
     });
 
+    ['row', 'column'].forEach((area) => {
+        [false, true].forEach((useNative) => {
+            function getHeaderCellByText($area, text) {
+                return $area.find('table:not(.dx-pivot-grid-fake-table) td')
+                    .filter((_, el) => { return el.innerText.trim() === text; })
+                    .get(0);
+            }
+
+            function checkLeftTopVisibleHeaderCellTexts(pivotGrid, expected, message) {
+                const $rowsHeaderArea = pivotGrid.$element().find('.dx-pivotgrid-vertical-headers');
+                const $columnsHeaderArea = pivotGrid.$element().find('.dx-pivotgrid-horizontal-headers');
+
+                const rowsAreaRect = $rowsHeaderArea.get(0).getBoundingClientRect();
+                const columnsAreaRect = $columnsHeaderArea.get(0).getBoundingClientRect();
+
+                const expectedRowCellRect = getHeaderCellByText($rowsHeaderArea, expected.row.toString()).getBoundingClientRect();
+                const expectedColumnCellRect = getHeaderCellByText($columnsHeaderArea, expected.column.toString()).getBoundingClientRect();
```

Comment:
Can we use 'elementFromPoint' instead?
Store {top, left} of the scrollable area and use elementFromPoint to get {top + 1, left + 1} to get the top/left visible cell:
document.elementFromPoint($('.dx-area-data-cell')[0].getBoundingClientRect().left + 1, $('.dx-area-data-cell')[0].getBoundingClientRect().top + 1).innerText - it's "$450" in ticket steps when "$2,410" is expected